app/assets/stylesheets/components/_form.scss

```diff
@@ -1,3 +1,112 @@
+.wizard-form {
+  text-align: center;
+
+  form {
+    border-width: 0;
+    width: 70%
+  }
+
+  label {
+    font-size: 25px;
+    margin-bottom: 5px;
+    color: $light-gray;
+  }
+
+  .art-form-text {
+    margin-top: 50px;
+    height: 50px;
+  }
+
+  .textarea-holder {
+    margin-bottom: 200px;
+
+  }
+
+  input {
+    height: 50px;
+    font-size: 20px;
+    border-color: $gold;
+    background-color: $main-grey;
+    color: white;
+    text-align: center;
+      ::placeholder {
+        color: $light-gray;
+      }
```

Comment:
This woule be
```
&::placeholder {
  color: $light-gray
  }
```
with a space between it and the text-align proprety but you shouldn't give the placeholder the same color as the text because it's confusing for the user.
The default color is fine, imo you can remove this